### PR TITLE
Add-on to introduction of README

### PR DESCRIPTION
**Pull request by rulep**
I have made an addition in clear, easy-to-understand language to the introduction. I believe my add-on makes the introduction even more comprehensive, while being free of complex technical terms. Seasoned developers, new programmers, and even nontechnical visitors will benefit from the clear language, exposing Peek to a broader range of users.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,12 +6,16 @@ Take a peek into your Rails application.
 
 ![Preview](https://f.cloud.github.com/assets/79995/244991/03cee1fa-8a74-11e2-8e33-283cf1298a60.png)
 
-This is a profiling tool originally built at GitHub to help us get an insight into our application. Now, we have extracted this into Peek, so that other Rails application can experience the same benefit.   
+No explanation of Peek is complete without discussing what Rails is. Rails is a framework written in the Ruby language for use in developing web applications.
```
Comment (jacobbednarz) [inline README.md]: Is it really a necessity to explain what Rails is? I'd think if you're using this gem, knowledge of what Rails does is expected.

File: README.md [hunk 2]
```diff
@@ -6,12 +6,16 @@ Take a peek into your Rails application.
...
 
 Peek puts a little bar on top of your application to show you all sorts of helpful information about your application. From the screenshot above, you can see that Peek provides information about database queries, cache, Resque workers and more. However, this is only part of Peek's beauty.
 
 The true beauty of Peek lies in the fact that it is an extensible platform. If there are some performance metrics that you need but are not available on Peek, you can find it from the list of available [Peek Views](#available-peek-views) and integrate it into Peek. Even if you do not find what you want on Peek Views, you can always [create your own](#creating-your-own-peek-item).
 
+A Peek Item is simply a class that will fetch and report the data to be displayed on the Peek bar. In order to do this, Peek keeps track of every request made by the Rails application you are building. By default, this information is stored in memory. To use Peek, it must be loaded into the environment. It can be enabled to see what’s going on by looking at the stats on the Peek bar. Peek has the option to display only the required info, so that your bar won’t get crowded.
```
Comment (jacobbednarz) [inline README.md]: I feel like this is just a rehash of what is said at https://github.com/peek/peek/pull/87/files#diff-04c6e90faac2675aa89e2176d2eec7d8R55 and https://github.com/peek/peek/pull/87/files#diff-04c6e90faac2675aa89e2176d2eec7d8R79. Are you able to point out what is different and how they differ?

Comment (rulep) [inline README.md]: My intent was to summarize the information for use by new programmers and nontechnical visitors wishing to learn more about the technology, prior to being engaged in the more technical aspects of it in the latter part of the README, but, yes, I can see and understand how its use requires knowledge of Rails, and that it even comes across as being repetitive. 

Thanks for the feedback, I appreciate it.

File: README.md (hunk 1 above)
Comment (dewski) [inline README.md]: I agree with @jacobbednarz. If this was an explanation that Peek is exclusively for Ruby on Rails (and not say Sinatra) that would be different.

Comment (rulep) [inline README.md]: @dewski, You make a good point, thanks!

**Comment by dewski** [comment]
@rulep thank you for taking the time to open this PR and revise the README, I'm going to leave what is in the current README as I believe it succinctly explains the purpose of peek and how to use/extend it. I have a preference for shorter a README over a verbose README when possible.

**Pull request closed without merging**